minigui/board.ts

```diff
@@ -284,7 +284,7 @@ class ClickableBoard extends Board {
   drawImpl() {
     super.drawImpl();
     let p = this.enabled ? this.p : null;
-    this.ctx.canvas.style.cursor = p ? 'pointer' : null;
+    this.ctx.canvas.style.cursor = p ? 'pointer' : 'none';
```

Comment:
javascript already has enough null types :angry:

Reply:
aha, but you see, that is not javascript!  It's CSS!

(sobs quietly)

Reply:
oh god no.

So you're saying that before it was replacing `undefined` with `null` and now it's replacing `undefined` with `"none"`.

:sob:

Reply:
ya.  Typescript (correctly) noted that this css attribute could not be null.  I think what has happened is the type annotations were extended to these objects with the latest es2019 or whatever.  But i dunno.  Having more type checking seems good!

Reply:
*sigh* `'none'` is not actually correct since it will hide the cursor. The intent with assigning `null` was to remove the property value, thereby setting it back to the default.

Will typescript allow you to assign the empty string `''`?

Reply:
Yes, it does, and that works better.  I made the assumption that the intent was to hide the cursor, so that's my bad.  Fixed.